pyrelational/strategies/regression/abstract_regression_strategy.py

```diff
@@ -18,6 +19,8 @@ def __init__(self):
 
     def __call__(self, num_annotate: int, data_manager: DataManager, model: ModelManager) -> List[int]:
         output = self.train_and_infer(data_manager=data_manager, model=model)
-        scores = self.scoring_fn(x=output.squeeze(-1))
+        if isinstance(output, Tensor):
```

Comment:
in what instances is it not a Tensor?

Reply:
When we create a GP model or even a Bayesian neural network, the output can be a torch.distribution that contains the mean and the std. Depending on the strategy, we can use both. But a torch.distribution cannot be squeezed.

Reply:
ah, of course! thanks for spotting it! There's a probably a few places in the strategies where this is a problem. We really need to handle multi-prediction then we wouldn't have to worry about this...